server/pkg/explorer/client.go

```diff
@@ -0,0 +1,68 @@
+package explorer
+
+import (
+	proxy "github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/client"
+	proxyTypes "github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/types"
+)
+
+var (
+	endpoints = map[string]string{
+		"main": "https://gridproxy.grid.tf",
+		"test": "https://gridproxy.test.grid.tf",
+		"qa":   "https://gridproxy.qa.grid.tf",
+		"dev":  "https://gridproxy.dev.grid.tf",
+	}
+)
+
+type ExplorerClient struct {
+	proxyClient proxy.Client
+}
+
+func NewClient() *ExplorerClient {
+	return &ExplorerClient{}
+}
+
+func (c *ExplorerClient) Load(net string) error {
```

Comment:
this function indicates it returns an error, but in the function body and error is never returned. Either validate the network input & return an error or drop the error from the function signature

Reply:
yes, forget to edit the function signature. it should set the network to `dev` as default so no need to return error. will remove it.